Doc/library/typing.rst

```diff
@@ -2815,6 +2815,56 @@ ABCs for working with IO
    represent the types of I/O streams such as returned by
    :func:`open`.
 
+The following protocols offer a simpler alternative for common use cases. They
+are especially useful for annotating function and method arguments and are
+decorated with :func:`@runtime_checkable <runtime_checkable>`.
+
+.. class:: Reader[T]
```

Comment:
These both need `.. versionadded:: 3.14`.

Reply:
Or rather `.. versionadded:: next` now, if I'm not mistaken (cf [the devguide](https://devguide.python.org/documentation/markup/#paragraph-level-markup))